examples/react-redux/js/components/AddTodo.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { PropTypes } from 'react';
+
+export default class AddTodo extends React.Component {
+
+	static propTypes = {
```

Comment:
Check http://stackoverflow.com/questions/38363156/static-proptypes-not-working-under-es6 -- is the `static` usage valid?